inc/TRestGeant4ParticleCollection.h

```diff
@@ -13,23 +13,22 @@
 ///
 //////////////////////////////////////////////////////////////////////////
 
-#ifndef RestCore_TRestGeant4ParticleCollection
-#define RestCore_TRestGeant4ParticleCollection
-
-#include <iostream>
-
-#include "TObject.h"
+#ifndef RestGeant4Lib_TRestGeant4ParticleCollection
+#define RestGeant4Lib_TRestGeant4ParticleCollection
 
+#include <TObject.h>
 #include <TRestGeant4Particle.h>
 
+#include <iostream>
+
 class TRestGeant4ParticleCollection : public TObject {
    protected:
     std::vector<TRestGeant4Particle> fParticles;
 
    public:
     static TRestGeant4ParticleCollection* instantiate(std::string model = "");
 
-    virtual void SetParticleModel(std::string modelstring) {
+    virtual void SetParticleModel(std::string modelString) {
         std::cout << "REST ERROR: SetParticleModel() called in base class "
```

Comment:
modelstring can be const? E.g. const std::string & modelString
Moreover, if the virtual function is not meant to be called in the base class you should do:
virtual void SetParticleModel(std::string modelString) = 0